src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+logging.pattern.dateformat=yyyy-MM-dd HH:mm:ss.SSS
+management.endpoints.jmx.exposure.include=*
```

Comment:
Can these be moved to the application YAML file, if so, can we also remove application.properties completely?

Reply:
Silly me, I remember us discussing this because it's auto generated from the tool I was using and then I just forgot to change it, you're right. Will change this.